content/updates/2020-08/2-API-search.md

```diff
@@ -0,0 +1,190 @@
+---
+pim_announcement_img: ./img/api-filters.png
+pim_announcement_alt_img: "API documentation about attribute filters"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# New updates to ease API catalog structure synchronization
+::: meta-data type="New" features="API, Connectivity" available="late August" link-to-doc="https://api.akeneo.com/documentation/filter.html"
+:::
+
+A **new `search` parameter** enables you to add filters to catalog structure endpoints. It will help reduce the number of data to process and the size of the API call responses. Moreover, we added **attribute group labels** inside the attribute endpoints to help you reduce the number of API calls needed to synchronize your catalog structure. 
+
+## The `search` parameter  
+
+Here are the available filters:
+- **updated date**: allows you to retrieve only data updated since a specific date and hour (available for attribute groups, attributes, and families), 
```

Comment:
```suggestion
- **updated date**: allows you to retrieve only data updated since a specific date and hour (available for attribute groups, and families), 
```